pkg/eth/node/node.go

```diff
@@ -119,27 +117,25 @@ func (reader PropertiesReader) GenesisBlock() string {
 	return header.Hash().Hex()
 }
 
-func (reader PropertiesReader) NodeInfo() (string, string) {
-	var info p2p.NodeInfo
-	err := reader.client.CallContext(context.Background(), &info, "admin_nodeInfo")
+func (reader PropertiesReader) NodeClientName() string {
+	var nodeName string
+	err := reader.client.CallContext(context.Background(), &nodeName, "web3_clientVersion")
 	if err != nil {
-		logrus.Debugf("error getting admin_nodeInfo: %s", err.Error())
+		logrus.Debugf("error getting client version: %s", err.Error())
 	}
-	return info.ID, info.Name
+	return nodeName
 }
 
-func (client ParityClient) NodeInfo() (string, string) {
-	nodeInfo := client.parityNodeInfo()
-	id := client.parityID()
```

Comment:
Should we also delete the `parityID` function?

Reply:
👍 nice catch!